tools/cldr-apps/js/src/index.js

```diff
@@ -12,9 +12,20 @@ import "ant-design-vue/dist/antd.css";
 import { showPanel, createCldrApp } from "./cldrVueRouter.js";
 import { runGui } from "./runGui.js";
 
+/**
+ * TODO Does not belong here. CLDR-14193
+ * Workaround (aka hack) due to flattening in the current info panel.
+ */
+function toggleTranscript() {
+  document
+    .getElementsByClassName("transcript-container")[0]
+    .classList.toggle("visible");
+}
+
 // The following will show up in the cldrBundle global
 export default {
   createCldrApp,
   runGui,
   showPanel,
+  toggleTranscript,
```

Comment:
@btangmu  yes, this function seems out of place in our top level exported bundle.  However, it seemed like the simplest/expedient way to add a toggle  accessible via a simple `a href="javascript:cldrBundle.toggleTranscript()"` since all javascript/events gets stripped out of the info panel until it gets rewritten.

Reply:
okay temporarily -- ideally we shouldn't need to refer to cldrBundle from inside any module

to get started modernizing info panel, the following dom elements should just be created once, and then never recreated from scratch, but the individual divs can be shown/hidden/cleared/repopulated individually, so that updating one doesn't have to destroy the others:

	itemInfo
		alertDiv
		voteDiv
		sidewaysDiv
		forumDiv
		xpathDiv

something like that is needed for https://unicode-org.atlassian.net/browse/CLDR-14045, and it will pave the way towards including vue components inside any of those divs